### PR TITLE
Issuer and verifier QR code display

### PR DESCRIPTION
**Pull request by sgreenbury**

### CODE REVIEW

File: trustchain-http/src/config.rs [hunk 1]
```diff
@@ -15,18 +15,23 @@ const DEFAULT_PORT: u16 = 8081;
 /// HTTP configuration.
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub struct HTTPConfig {
-    /// Hostname for server
+    /// Host address for server. For example, Android emulator `10.0.2.2` refers to `127.0.0.1` of
```
Comment (sgreenbury) [inline trustchain-http/src/config.rs:18]:
```suggestion
    /// Host address for server.
```

File: trustchain-http/src/config.rs [hunk 2]
```diff
@@ -15,18 +15,23 @@ const DEFAULT_PORT: u16 = 8081;
...
+    /// machine running emulator.
     pub host: IpAddr,
-    /// Hostname reference. For example, Android emulator 10.0.2.2 refers to 127.0.0.1 of machine running emulator.
+    /// Hostname reference. For example, Android emulator 10.0.2.2 refers to 127.0.0.1 of machine
```
Comment (sgreenbury) [inline trustchain-http/src/config.rs:21]:
```suggestion
    /// Hostname display in QR codes. For example, if using local server with an Android emulator
    /// `10.0.2.2` refers to `127.0.0.1` of machine
```

File: trustchain-http/src/qrcode.rs [hunk 3]
```diff
@@ -3,6 +3,16 @@ use base64::write::EncoderWriter;
 use image::Luma;
 use image::{DynamicImage, ImageOutputFormat};
 use qrcode::QrCode;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+/// QR code JSON type of verifiable content.
+pub(crate) struct DIDQRCode {
```
Comment (sgreenbury) [inline trustchain-http/src/qrcode.rs:11]: Add doc comment for fields

Comment (sgreenbury) [reply trustchain-http/src/qrcode.rs:11]: Completed

**Review by thobson88 — APPROVED** [review]
LGTM!

**Pull request merged**